### ISSUE TITLE
Add support of vueIndentScriptAndStyle to format-query-block

### ISSUE THREAD

**Issue by zgmrvn** [gridsome/eslint-plugin-gridsome#60]
Add support of [vueIndentScriptAndStyle](https://prettier.io/docs/en/options.html#vue-files-script-and-style-tags-indentation) for `<static-query>` and `<page-query>`. This rule let you ignore the first level of indentation in `<script>` and `<style>` tags and seem not to be respected by the `format-query-block` rule.

What should be possible:
```vue
<static-query>
query {
  metadata {
    siteName
  }
}
</static-query>
```
What is enforced by the `format-query-block`:
```vue
<static-query>
  query {
    metadata {
      siteName
    }
  }
</static-query>
```

**Comment by tyankatsu0105** [comment]
@zgmrvn
Thanks for the reporting issue!
Currently, `format-query-block` rule looks at the `.prettierrc` file.
So, probably this feature is easy.
I'll work :)

**Comment by tyankatsu0105** [comment]
Hi @zgmrvn 
I released v1.5.0
We can set option `overridePrettierOption` at this version.
Thank you for the proposal.

You can join all-contributors of eslint-plugin-gridsome if you would not mind 
See: https://github.com/gridsome/eslint-plugin-gridsome/blob/master/CONTRIBUTING.md#did-you-contribute-to-eslint-plugin-gridsome

**Comment by zgmrvn** [comment]
👌 perfect
